.bonsai.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
I'm wondering if we could maybe have something similar to rubocop that allows us to pull in common attributes from a single location so its quick to update.

Reply:
hmm.  Interesting.  We could have the sensu-go-bonsai-asset repo as a submodule or subtree.

Reply:
I'll look at this as a future enhancement.  One problem here is that bonsai currently needs the .bonsai.yml to exist in the master branch to register the repository. So I'm not sure we can automate this completely.